docs/loaders.md

```diff
@@ -145,12 +146,13 @@ For example, for the file `quakes.csv`, the following data loaders are considere
 - `quakes.csv.R`
 - `quakes.csv.rs`
 - `quakes.csv.go`
+- `quakes.csv.java`
 - `quakes.csv.sh`
 - `quakes.csv.exe`
 
-If you use `.py`, `.R`, `.rs`, or `.go`, the corresponding interpreter (`python3`, `Rscript`, `rust-script`, or `go run`, respectively) must be installed and available on your `$PATH`. Any additional modules, packages, libraries, _etc._, must also be installed before you can use them.
+If you use `.py`, `.R`, `.rs`, `.go` or `java`, the corresponding interpreter (`python3`, `Rscript`, `rust-script`, `go run` or `java`, respectively) must be installed and available on your `$PATH`. Any additional modules, packages, libraries, _etc._, must also be installed before you can use them.
 
-Whereas `.js`, `.ts`, `.py`, `.R`, `.rs`, `.go`, and `.sh` data loaders are run via interpreters, `.exe` data loaders are run directly and must have the executable bit set. This is typically done via [`chmod`](https://en.wikipedia.org/wiki/Chmod). For example:
+Whereas `.js`, `.ts`, `.py`, `.R`, `.rs`, `.go`, `.java` and `.sh` data loaders are run via interpreters, `.exe` data loaders are run directly and must have the executable bit set. This is typically done via [`chmod`](https://en.wikipedia.org/wiki/Chmod). For example:
```

Comment:
```suggestion
Whereas all other data loaders are run via interpreters (such as `node` for `.js`), `.exe` data loaders are run directly and must have the executable bit set. This is typically done via [`chmod`](https://en.wikipedia.org/wiki/Chmod). For example:
```

I think we have enough data loaders now that it makes sense to refer to them in aggregate and only call out `.exe` is being special.